kong.yaml

```diff
@@ -25,8 +25,8 @@ _transform: true
 
 services:
 - name: mps-rest
-  host: mps
-  port: 3000
+  host: mpslookup
```

Comment:
We'll need to figure out the best way to do this---this is only for swarm , it is not needed for compose-- but maybe we just always have it...thoughts?

Reply:
Can I revert the changes back to mps and 3000 port? Document the steps to modify the kong.yaml, if it needs to deploy in docker swarm.